es/chromebook_setup/instructions.md

```diff
@@ -8,7 +8,7 @@ Nube de IDE es una herramienta que te ofrece un editor de código y el acceso po
 
 1. Ve a [PaizaCloud Cloud IDE](https://paiza.cloud/)
 2. Crea una cuenta
-3. Haz clic en *Nuevo Servidor* y elige la aplicación Django
+3. Haga clic en *Nuevo Servidor* y elija la aplicación Django
```

Comment:
All the other parts of this tutorial are using the same forma 2nd person so I don't understand why this was change!?